### DISCUSSION TITLE
compilation error

### DISCUSSION THREAD

**Discussion by Miguel-azevedo9** [joaomlourenco/novathesis#474] [Q&A]
Professor,

I used the template to submit my thesis, I finished it on the 30th of September. Now, after the thesis defense I need to make some minor changes so I tried to open the latex document on Overleaf. I used the same document that I compiled in September but now for some reason, I cannot see the compiled document and I have error messages. I used the version 7.1.5. I don't know why this is happening and would like some help in order to submit the final version. 

Thank you in advance 

Miguel Azevedo

**Comment by joaomlourenco** [comment]
My dear Miguel.
And you expect that I guess what is your problem from "_I have error messages_"?

**Reply by Miguel-azevedo9** [reply]
It is indeed quite a task, I admit.  I was just surprised that errors appeared now in the same version that was okay two months ago. here are the errors that I now have:

The first one "Package biblatex Error: Biber backend is required by style."  I tried to uncomment biber and bibtex and i still have the error.

the second one "LaTeX Error: Option clash for package xcolor."

the third one "LaTeX Error: There's no line here to end." 
here in this quote in quote.tex

\begin{ntquote}[8cm][Galileo][Somewhere in a book or speach][Astronomer, physicist and engineer][en]
  You cannot teach a man anything; you can only help him discover it in himself.%
\end{ntquote}

LaTeX Error: There's no line here to end. (at the apendix1.tex)

\chapter{}%\emph{NOVAthesis} covers showcase}

These are the main ones.
Thank you in advance

**Reply by Miguel-azevedo9** [reply]
professor, I can always provide more details or share my document if that helps to solve the errors.
Thank you

**Reply by joaomlourenco** [reply]
Miguel,

I would suggest for you to start from a brand new version of the template, and add your files _one-by-one_, to understand where is your problem.

Otherwise I would recommend you to hire the services of a LaTeX expert to help you diagnosing and fixing the problem.

**Reply by Miguel-azevedo9** [reply]
Professor, I tried to do as suggested. I downloaded a new template version 7.1.12, and uploaded it to Overleaf, where I wrote my entire thesis, and now I cannot even compile the original template. I just get these errors. I don't know if it's something specific with overleaf but right now the original template is not compiling correctly. I'm trying to figure out what all the errors are but I could really use your help. I need to submit my thesis. The \label error, line 145 is the one I'm having the most trouble with.


Package nameref Warning: The definition of \label has changed! Check your packages! Replacing it with the kernel definition on input line 145.

 
‪/usr/local/texlive/2023/texmf-dist/tex/latex/hyperref/nameref.sty, 145‬
Package hyperref Warning: Token not allowed in a PDF string (Unicode): removing `\includegraphics' on input line 31.

 
‪./Chapters/chapter1.tex, 31‬
Package hyperref Warning: Token not allowed in a PDF string (Unicode): removing `\fontcharht' on input line 31.

 
‪./Chapters/chapter1.tex, 31‬
Package hyperref Warning: Token not allowed in a PDF string (Unicode): removing `\font' on input line 31.

Thank you
